core/src/trezor/ui/layouts/tr/__init__.py

```diff
@@ -1067,6 +1067,56 @@ async def confirm_total(
     )
 
 
+async def confirm_ethereum_staking_tx(
```

Comment:
Looks like `confirm_ethereum_staking_tx` in `TR` and `TT` code have now different signature --- the position of `chunkify` differs. Also, why is the value of `chunkify` not used in `TR`'s case?

Reply:
`chunkify` is used on TS3. I also found that `confirm_value` can be used in TT with just additional params.
https://github.com/trezor/trezor-firmware/pull/3534/commits/4b4d1fadd319297eb89c383b45fc509650624c0e